Peoplesoft/main.tf

```diff
@@ -0,0 +1,538 @@
+# Peoplesoft/main.tf
+
+locals {
+    subnet_bits = 8   # want 256 entries per subnet.
+    # determine difference between VNET CIDR bits and that size subnetBits.
+    vnet_cidr_increase = "${32 - element(split("/",var.vnet_cidr),1) - local.subnet_bits}"
+    subnetPrefixes = {
```

Comment:
need another subnet for identity. Maybe we should reduce the subnet_bits to 3 or something as we have 6 subnets.

Reply:
Added extra subnet.